README.md

```diff
@@ -8,14 +8,512 @@ A simple library to convert units. Heavily based on [convert-unitsss](https://gi
 [Example](https://appsonrent.com/wp-content/uploads/2017/07/Under-Cunstruction-Banner1600X900.jpg)
 
 ### Key Features
-* Tree shakable
-* No dependencies
-* Build for es6, cmj and umd
+* Tree shakable.
+* No dependencies.
+* Build for es6, cmj and umd.
+* Tested to 4 significant figures.
 
 ## Installation
 
+It will be released in npm and CDN during the first days of August.
+
 ## Usage
 
+The library provide a simple chained API to transform units of the same type.
+The units are broken down to modules, so it is possible and recommended, to load only the unite types that you are using.
+```js
+import { voltage } from '../src/index.js'
```

Comment:
In Angular it is enough to 
`import { voltage } from 'units-converter'

Reply:
done